llvm-general/src/LLVM/General/Internal/FFI/PassManagerC.cpp

```diff
@@ -85,8 +87,16 @@ void LLVM_General_AddGlobalValueNumberingPass(LLVMPassManagerRef PM, LLVMBool no
 }
 
 void LLVM_General_AddInternalizePass(LLVMPassManagerRef PM, unsigned nExports, const char **exports) {
-	std::vector<const char *> exportList(exports, exports + nExports);
-	unwrap(PM)->add(createInternalizePass(exportList));
+    const std::vector<const char *> exportList(exports, exports + nExports);
```

Comment:
No need to allocate/free a vector any more. Just iterate over the array in the lambda.